src/qnngds/devices/snspd.py

```diff
@@ -4,40 +4,100 @@
 
 import phidl.geometry as pg
 from typing import Tuple, Optional, Union
+import qnngds._default_param as dflt
 
 
-def vertical(
+def basic(
     wire_width: float = 0.2,
     wire_pitch: float = 0.6,
     size: Tuple[Union[int, float], Union[int, float]] = (6, 10),
     num_squares: Optional[int] = None,
+    turn_ratio: Union[int, float] = 4,
     terminals_same_side: bool = False,
+    layer: int = dflt.layers["device"],
+) -> Device:
+    """Creates an optimally-rounded SNSPD.
```

Comment:
I don't understand the difference between the two functions (`basic` and `vertical`)
Maybe explain more or give a use case in the description?